Source/WebCore/platform/ScrollTypes.h

```diff
@@ -341,35 +341,30 @@ enum class ScrollSnapPointSelectionMethod : uint8_t {
     Closest,
 };
 
+enum class ScrollClass : uint8_t {
```

Comment:
Maybe `ScrollType` ?

Reply:
Unfortunately `ScrollType` is being used for differentiating user vs programatic scroll.